docs/setup/installation.md

```diff
@@ -47,8 +47,10 @@ For more details, see
 
 ##### Matrix.org packages
 
-Matrix.org provides Debian/Ubuntu packages of Synapse via
-<https://packages.matrix.org/debian/>.  To install the latest release:
+Matrix.org provides Debian/Ubuntu packages of Synapse, for the amd64
```

Comment:
Don't we also do this for arm?

Reply:
The answer seems to be no: https://packages.matrix.org/debian/dists/hirsute/main/.
(Some old distros have i386 too, but that appears to have stopped in July. I don't know if this was intentional or not?)

Reply:
I was thinking of https://github.com/matrix-org/synapse/blob/develop/.github/workflows/docker.yml#L72 but that's a docker container rather than a debian package

Reply:
cf https://github.com/matrix-org/synapse/issues/1496

> Some distros have i386 too, but that appears to have stopped in July.

It was a long time before that. We stopped doing i386 packages when we dropped the python2 packages, back in January 2019.